spec/02-integration/03-admin_api/09-targets_routes_spec.lua

```diff
@@ -257,6 +257,65 @@ describe("Admin API", function()
           assert.equal([[{"data":[],"total":0}]], body)
         end)
       end)
+
+      describe("active targets", function()
```

Comment:
Admin API tests should ideally be scoped under the endpoint they are testing, in this case, we should ideally have:
```
describe("Admin API", function()
  describe("/upstreams/{upstream}/targets/active", function()
    it("only shows active targets", function()

    end)
  end)
end)
```

Hence this test in busted will be:

```
Admin API /upstreams/{upstream}/targets/active only shows active targets
```